cli/commands/catalog/tui/models/list/model_keys.go

```diff
@@ -0,0 +1,72 @@
+package list
+
+import (
+	"github.com/charmbracelet/bubbles/key"
+	"github.com/charmbracelet/bubbles/list"
+)
+
+// NewKeyMap returns a set of keybindings.
+func NewKeyMap() list.KeyMap {
+	return list.KeyMap{
+		// Browsing.
+		CursorUp: key.NewBinding(
+			key.WithKeys("up", "ctrl+p"),
+			key.WithHelp("↑/ctrl+p", "move up"),
+		),
+		CursorDown: key.NewBinding(
+			key.WithKeys("down", "ctrl+n"),
+			key.WithHelp("↓/ctrl+n", "move down"),
+		),
+		PrevPage: key.NewBinding(
+			key.WithKeys("left", "pgup", "alt+v"),
+			key.WithHelp("←/pgup/alt+v", "prev page"),
+		),
+		NextPage: key.NewBinding(
+			key.WithKeys("right", "pgdown", "ctrl+v"),
+			key.WithHelp("→/pgdn/ctrl+v", "next page"),
+		),
+		GoToStart: key.NewBinding(
+			key.WithKeys("home", "ctrl+a"),
+			key.WithHelp("home/ctrl+a", "go to start"),
+		),
+		GoToEnd: key.NewBinding(
+			key.WithKeys("end", "ctrl+e"),
+			key.WithHelp("end/ctrl+e", "go to end"),
+		),
+		Filter: key.NewBinding(
+			key.WithKeys("/"),
+			key.WithHelp("/", "filter"),
```

Comment:
Maybe "filter / search"? I think some folks will be looking for the word "search."

Reply:
> Maybe "filter / search"? I think some folks will be looking for the word "search."

It's better to set only one of them, otherwise `/` can be confused with a hotkey

![1  tmuxinator local 2023-12-07 at 10 33 29 PM](https://github.com/gruntwork-io/terragrunt/assets/62853952/1f67b845-1963-4468-90f6-92e0da843bf7)

I renamed to "search", I agreed it's clearer.


> Also, what content is getting indexed? Is it just the README for each module? The module name and folder name? Something else? And is it doing exact matches? Or fuzzy matching? Case sensitive?

Only the module name is indexed with fuzzy and case-insensitive matching.

![1  tmuxinator local 2023-12-07 at 10 42 40 PM](https://github.com/gruntwork-io/terragrunt/assets/62853952/cd8c0aea-ece7-48a9-b5cc-546ed4641e97)

Reply:
Roger!

Could you add a follow-up ticket to index the module README as well?

Reply:
Sure, created the GH issue https://github.com/gruntwork-io/terragrunt/issues/2833.